packages/chakra-ui/src/AltDateWidget/AltDateWidget.tsx

```diff
@@ -129,7 +129,7 @@ function AltDateWidget<
 
   return (
     <Box>
-      <Box display="flex" flexWrap="wrap" alignItems="center" justify="center">
+      <Flex flexWrap="wrap" alignItems="center">
```

Comment:
I think I happened to pull in a more recent version of chakra-ui and discovered `justify` is not a valid prop on `Box`. We can use the `Flex` component, instead. I think we actually want to keep this left-aligned, though, so I ended up removing `justify="center"` anyways.